discord/enums.py

```diff
@@ -188,6 +189,106 @@ def try_value(cls, value):
             except (KeyError, TypeError):
                 return value
 
+# Thanks to Dest0re on GitHub for providing this enum!
+class EventType(Enum):
```

Comment:
Maybe make different categories for these and adjust the values similar to AuditLogEvent

Reply:
I'm not sure about doing this. I'll come back to this later.

Reply:
I came back to this idea and I've decided why I don't like it. It kinda clutters up the enums file with something that could just be combined into one enum.